src/apps/components/AppAlerts/useAllAppsAlert.ts

```diff
@@ -30,13 +52,11 @@ export const useAllAppsAlert = (): FailedWebhooksCount => {
         const webhookInfo = defaultFailedWebhooksInfo;
 
         app.node.webhooks?.forEach(webhook => {
-          const { failedDelivers, pendingDelivers } = webhook;
-
-          if (failedDelivers && failedDelivers.edges?.length > 0) {
+          if (hasFailedCheck(webhook)) {
             webhookInfo.hasFailed = true;
           }
 
-          if (pendingDelivers && pendingDelivers.edges?.length > 0) {
+          if (hasPendingCheck(webhook)) {
             webhookInfo.hasPending = true;
```

Comment:
it's not hasPending anymore - the flag is set when we have at least one pending attempt